src/main/scala-sbt-1.0/scalafix/sbt/ScalafixTestkitPlugin.scala

```diff
@@ -13,38 +13,42 @@ object ScalafixTestkitPlugin extends AutoPlugin {
   override def requires: Plugins = JvmPlugin
 
   object autoImport {
-    case class InputAxis(scalaVersion: String) extends VirtualAxis.WeakAxis {
+    case class TestkitTargetAxis(scalaVersion: String)
+        extends VirtualAxis.WeakAxis {
       private val scalaBinaryVersion =
         CrossVersion.binaryScalaVersion(scalaVersion)
 
-      override val idSuffix = s"Input${scalaBinaryVersion.replace('.', '_')}"
-      override val directorySuffix = s"input$scalaBinaryVersion"
+      override val idSuffix = s"Target${scalaBinaryVersion.replace('.', '_')}"
+      override val directorySuffix = s"target$scalaBinaryVersion"
     }
 
-    object InputAxis {
-      def inputScalaVersion(virtualAxes: Seq[VirtualAxis]): String =
-        virtualAxes.collectFirst { case a: InputAxis => a.scalaVersion }.get
-    }
+    object TestkitTargetAxis {
 
-    def resolveByInputAxis[T](
-        matrix: ProjectMatrix,
-        key: TaskKey[T]
-    ): Def.Initialize[Task[T]] =
-      Def.taskDyn {
-        val sv = InputAxis.inputScalaVersion(virtualAxes.value)
-        val project = matrix.finder().apply(sv)
-        Def.task((project / key).value)
-      }
+      private def targetScalaVersion(virtualAxes: Seq[VirtualAxis]): String =
```

Comment:
I don't completely get why we take the first scalaVersion that we find, and use it the resolve task

Reply:
> why we take the first scalaVersion that we find

because there can be only one value per axis per project

> use it the resolve task

we lookup the corresponding input/output project based on this value, that's where I am not too satisfied with the `Target` naming: it's about tagging in `tests*` which `input*` or `output*` it should run against.